Azure/azure-cli
# [Network] `az network bastion create`: Add `--tags` argument

Description

Add `--tags` argument to az network bastion create.

Testing Guide

`network bastion create -g {rg} -n {bastion} --vnet-name {vnet} --public-ip-address {ip} --tags a=b`

---

This checklist is used to make sure that common guidelines for a pull request are followed.
